consumatio/external/tmdb.py

```diff
@@ -11,61 +11,63 @@
 from consumatio.gateways.episode_images_to_dict import *
 from consumatio.external.db import Database
 import json
+import os
 
 class Tmdb():
 
     def __init__(self):
         self.db = Database()
+        self.tmdb_key = os.getenv('TMDB_KEY')
```

Comment:
While this works, shouldn't we decompose this to the constructor? Otherwise, testing would involve setting an env var.

Reply:
I agree. But is there any other option? You need a key to make requests and if you want to use integration tests, we certainly will need to set a key somewhere.

Reply:
Yeah, just put it in the constructor and supply it in `main` :)

Reply:
Agreed. Please look at my new changes and see if it's sufficient. I didn't want to modify too much for this to work.